src/stuf/stuf_bg/templates/stuf_bg/StufBgRequest.xml

```diff
@@ -1,5 +1,6 @@
-{% extends "stuf/soap_envelope.xml" %}{% load stuf %}
+{% extends "stuf/soap_envelope.xml" %}{% load stuf l10n %}
 {% block body %}
+{% localize off %}
```

Comment:
is it possible to put this outside the `block body`? That way child templates overriding the block body don't need to remember overriding the `localize` tag.